validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/OwnedValidators.java

```diff
@@ -58,6 +61,12 @@ public int getValidatorCount() {
     return Optional.ofNullable(validators.get(publicKey));
   }
 
+  public List<ActiveValidator> getActiveValidators() {
+    return validators.values().stream()
+        .map(validator -> new ActiveValidator(validator.getPublicKey(), validator.isReadOnly()))
+        .collect(Collectors.toList());
+  }
```

Comment:
Given `Validator` has all the information we need I think we just remove `ActiveValidator` and use `Validator` directly.

Then this method would just be `return unmodifiableSet(validators.values())`.  Would mean changing a few serializers in the rest api code but that's fine - we've learnt more as we implemented so fine for things to change to reflect that new knowledge.